static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -24,6 +25,7 @@ type Props = {
   event: Event;
   platform: PlatformType;
   className?: string;
+  debugFrames?: StacktraceFilenameQuery[];
```

Comment:
```suggestion
  debugFrames?: StackTraceFilenameQuery[];
```
minor: shall we name it Stack**T**race...?